src/Cmf/MainBundle/Resources/data/posts/2015-04-01.yml

```diff
@@ -0,0 +1,62 @@
+name: "cmf-hackday-may-2015"
+parent: "/news"
+title: "CMF Hackday 1st/2nd May 2015"
+label: false
+format: markdown
+template: CmfMainBundle:Cms:news_detail.html.twig
+publish_start_date: "2015-04-01"
```

Comment:
lets change to 2015-04-02 if we publish not today (avoid aprils fool confusion, too)